File: 0283-move-zeroes/0283-move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        count = 0
        i=0
        while i != len(nums):
            if nums[i] == 0:
                count += 1
                nums.remove(0)
            else:
                i += 1
        for _ in range(count):
            nums.append(0)

File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int nonZeroIndex = 0; // Index to place the next non-zero element
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != 0) {
                // Swap non-zero element with the element at nonZeroIndex
                int temp = nums[nonZeroIndex];
                nums[nonZeroIndex] = nums[i];
                nums[i] = temp;
                nonZeroIndex++;
            }
        }
    }
}